drill-cicd-gradle-plugin/src/main/kotlin/com/epam/drill/integration/gradle/AgentRunnerTasks.kt

```diff
@@ -101,8 +101,8 @@ private fun AgentConfiguration.mapGeneralAgentProperties(
     this.logLevel = agentTaskExtension.logLevel ?: agentPluginExtension.logLevel
     this.logFile = (agentTaskExtension.logFile ?: agentPluginExtension.logFile)?.let { File(it) }
 
-    this.drillApiUrl = pluginExtension.drillApiUrl.fromEnv("DRILL_API_URL").required("drillApiUrl")
-    this.drillApiKey = pluginExtension.drillApiKey.fromEnv("DRILL_API_KEY")
+    this.apiUrl = pluginExtension.apiUrl.fromEnv("API_URL").required("apiUrl")
```

Comment:
I would stay `DRILL` prefix in environment variables:
 `.fromEnv("DRILL_API_URL")`

Applies here and in similar places below.